app/src/main/res/values-de/strings.xml

```diff
@@ -276,7 +276,7 @@
     <string name="loading">Wird geladen</string>
     <string name="loading_failed">Ladevorgang fehlgeschlagen</string>
     <string name="settings_dialog_url_open_title">Keine kompatible App gefunden</string>
-    <string name="settings_dialog_url_open_text">BetterUntis konnte keine App finden, die mit dem Öffnen von Links umgehen kann.
+    <string name="settings_dialog_url_open_text">BetterUntis konnte keine App finden, die mit dem Links öffnen kann.
```

Comment:
This sentence is grammaticly not correct. Please provide a valid german sentence.

Reply:
Sorry, i mean "BetterUntis konnte keine App finden, die mit Links öffnen kann."

Reply:
I would prefer: "BetterUntis konnte keine App finden, die Links öffnen kann."

Reply:
My device make (whyever) autocorrect. Your Version is better.